pandas/core/ops/docstrings.py

```diff
@@ -233,6 +233,61 @@ def _make_flex_doc(op_name, typ):
 dtype: float64
 """
 
+_eq_example_SERIES = """
+Examples
+--------
+>>>s1=pd.Series((1,2,3,4,5))
```

Comment:
These are going to need a space after the ">>>" and after each comma.  Also pls pass a list to Series instead of tuple

Reply:
hey :)
I saw these >>> in the code, what were they meant to mark?

Reply:
That's what it looks like if you run the code in the python interpreter